### DISCUSSION TITLE
Strapi information about update

### DISCUSSION THREAD

**Discussion by CtFelix** [strapi/strapi#7179] [Legacy Topics]
Hi
Where i can find information about strapi updating?
What was added or changed in new version?

**Comment by MattieBelt** [comment]
Hi @CtFelix, take a look at all the releases.
https://github.com/strapi/strapi/releases

**Reply by lauriejim** [reply]
And here are migration guides if needed - https://strapi.io/documentation/v3.x/migration-guide/#guides

**Reply by CtFelix** [reply]
@MattieBelt, thanks, this is what i was looking for
@lauriejim thank you
